examples/rws.py

```diff
@@ -0,0 +1,66 @@
+from datetime import date
```

Comment:
Why is this example needed?

I was hoping we'd remove all examples as soon as we have good test coverage, so I'd rather not add another file here.

Reply:
I copied it to my notes to add these tests to their appropriate files, and forgot to remove it. Good catch!

Reply:
This will be removed before merge.